qemu/tests/cfg/drive_mirror.cfg

```diff
@@ -56,6 +56,21 @@
                         - block_job_complete:
                            type = drive_mirror_complete
                            variants:
+                               - granularity:
+                                   when_steady = check_granularity
```

Comment:
Let's re-factor this configuration file:

- drive_mirror:
      variants:
            - 2qcow2:
                  image_format_target = qcow2
            - 2raw:
                  image_format_target = raw
     variants:
           - @on_localfs:
           - on_nfs:
           - on_iscsi:
 variants:
           - job_complete:
           - job_cancel:
           - query_job_status:
           - set_job_speed:
           - reset_job_speed:
           - continuous_backup:
           - with_reboot:
           - with_stress:

And, move native tests to a special test loop, name like drive_mirror_negative_tests.cfg